strongbox-web-core/src/test/java/org/carlspring/strongbox/rest/UserControllerTest.java

```diff
@@ -0,0 +1,227 @@
+                                            .when()
+                                            .put("/users/user")
+                                            .peek() // Use peek() to print the ouput
+                                            .then()
+                                            .statusCode(200) // check http status code
+                                            .extract()
+                                            .asString();
+
+        System.out.println(response);
+
+        // deserialize response
+        User updatedUser = objectMapper.readValue(response, User.class);
+
+        assertEquals(createdUser, updatedUser);
+
+    }
+
+    @Test
+    @WithUserDetails("admin")
+    public void testDeleteUser()
+            throws Exception
+    {
+        // create new user
+        User test = buildUser("test-update", "password-update");
+
+        RestAssuredMockMvc.given()
+                          .contentType("application/json")
+                          .param("juser", test)
+                          .when()
+                          .post("/users/user")
+                          .peek() // Use peek() to print the ouput
+                          .then()
+                          .statusCode(200) // check http status code
+                          .extract()
+                          .asString();
+
+        RestAssuredMockMvc.given()
+                          .contentType("application/json")
+                          .param("The name of the user", test.getUsername())
+                          .when()
+                          .delete("/users/user/" + test.getUsername())
+                          .peek() // Use peek() to print the ouput
+                          .then()
+                          .statusCode(200) // check http status code
+                          .extract()
+                          .asString();
+
+    }
+
+    private User retrieveUserByName(String name)
+            throws IOException
+    {
+        String response;
+
+        response = RestAssuredMockMvc.given()
+                                     .contentType("application/json")
+                                     .param("The name of the user", name)
+                                     .when()
+                                     .get("/users/user/" + name)
+                                     .peek() // Use peek() to print the ouput
+                                     .then()
+                                     .statusCode(200) // check http status code
+                                     .extract()
+                                     .asString();
+
+        User admin = objectMapper.readValue(response, User.class);
+        return admin;
+    }
+
+    private User buildUser(String name,
+                           String password)
+    {
+        User test = new User();
+        test.setUsername(name);
+        test.setPassword(password);
+        test.setEnabled(false);
+        return test;
+    }
+
+}
```

Comment:
Please, add a new line at the end of this file and all others that do not have one (that you've edited/created).